### ISSUE TITLE
Inspired from POI?

### ISSUE THREAD

**Issue by tonmoyborah** [nwojke/deep_sort#97]
This is more of a question than an issue. I was going through both DeepSORT and POI papers and noticed some similarities. My question is, is DeepSORT a better(enhanced/optimised) algorithm over the POI implementation? If no, what is the basic difference? Also, the MOTA reported in your paper is more for POI than DeepSORT, is it strictly because of the huge training set and practices used for the detection phase? 
Thanks for the awesome repository btw.

**Comment by liangzimei** [comment]
i agree, when i use det.txt download from MOT website to generate detections, the acc of tracking is worse than using the POI detection npy files.

**Comment by derkbreeze** [comment]
Mainly because the POI used state of art person detections while det.txt downloaded from MOT is only using Faster RCNN.
I was thinking about deep_sort utilized stronger features for matching. e.g. features extracted from pretrained person reid model and utilized different distance for matching bounding boxes. And it can run in real time. @tonmoyborah
